src/guide/reactivity-computed-watchers.md

```diff
@@ -211,6 +213,39 @@ firstName.value = 'John' // logs: ["John", ""] ["", ""]
 lastName.value = 'Smith' // logs: ["John", "Smith"] ["John", ""]
 ```
 
+However, if you are changing both watched sources simultaneously in the same method, the watcher will be executed only once:
+
+```js{9-13}
+setup() {
+  const firstName = ref("");
+  const lastName = ref("");
+
+  watch([firstName, lastName], (newValues, prevValues) => {
+    console.log(newValues, prevValues);
+  });
+
+  const changeValues = () => {
+    firstName.value = "John";
+    lastName.value = "Smith";
+    // logs: ["John", "Smith"] ["", ""]
+  };
+
+  return { changeValues };
+}
+```
+
+Note that multiple synchronous changes will only trigger the watcher once.
+
+If you want the watcher to be triggered after every change, make it synchronous (`sync: true`, not recommmended), or make sure to wait a tick  between each change, i.e. by using [nextTick](/api/global-api.html#nexttick):
```

Comment:
I believe the setting is `flush: 'sync'` rather than `sync: true`. As requested, I've made an attempt at rewording this paragraph:

```suggestion
It is possible to force the watcher to trigger after every change by using the setting `flush: 'sync'`, though that isn't usually recommended. Alternatively, [nextTick](/api/global-api.html#nexttick) can be used to wait for the watcher to run before making further changes. e.g.:
```

Reply:
Oh right! Thanks!